src/couchapp/AsyncTransfer/views/LFNSiteByLastUpdate/map.js

```diff
@@ -7,13 +7,6 @@ function complete_job(doc, req) {
 
 function(doc) {
 	if(doc.lfn && complete_job(doc)){
```

Comment:
also why need additional function `complete_job`, while it can be changed to doc['state']  == 'done'